lib/bitcoin/interfaces/IFeeCalculator.ts

```diff
@@ -7,6 +7,8 @@ import BlockMetadataWithoutNormalizedFee from '../models/BlockMetadataWithoutNor
 export default interface IFeeCalculator {
   /**
    * Returns the fee in satoshis for a particular block height.
+   * @throws SidetreeError with ErrorCode.NormalizedFeeCalculatorBlockNotFound if the block is not observed by the
+   * BitcoinProcessor yet
```

Comment:
It's an interface, shouldn't know about `BitcoinProcessor`.
```suggestion
   * @throws SidetreeError with ErrorCode.NormalizedFeeCalculatorBlockNotFound if fee is not found for the given block.
```